### PR TITLE
stop adding stable helm repos by default

### LINKED ISSUE #359: should we disable adding stable/incubator helm repos by default?
Helm 3 [no longer sets repos by default](https://helm.sh/docs/topics/v2_v3_migration/#overview-of-helm-3-changes):

```
local or stable repositories are not set-up by default
```
Helmsman v3.0.0-betaX still adds those repos by default and we have  a flag to disable that behaviour. Should we reverse this (not add repos by default and provide a flag to add them) to be aligned with the helm 3 behaviour? Or the current behaviour is good enough? 

Any thoughts @luisdavim @mkubaczyk

### PR DESCRIPTION
**Pull request by sami-alajrami**
Fixes #359

### CODE REVIEW

File: examples/example.toml [hunk 1]
```diff
@@ -66,6 +66,8 @@ context= "test-infra"   # defaults to "default" if not provided
 [helmRepos]
   argo = "https://argoproj.github.io/argo-helm"
   jfrog = "https://charts.jfrog.io"
+  # stable = "https://kubernetes-charts.storage.googleapis.com"
+	# incubator = "http://storage.googleapis.com/kubernetes-charts-incubator"
```
Comment (luisdavim) [inline examples/example.toml]: nit: why not the same indentation?

File: internal/app/utils.go [hunk 2]
```diff
@@ -57,7 +57,7 @@ func fromTOML(file string, s *state) (bool, string) {
 	if _, err := toml.Decode(tomlFile, s); err != nil {
 		return false, err.Error()
 	}
-	addDefaultHelmRepos(s)
+	//addDefaultHelmRepos(s)
```
Comment (luisdavim) [inline internal/app/utils.go]: nit: if the function was removed there's no point in keeping this commented out here.

File: internal/app/utils.go [hunk 3]
```diff
@@ -109,7 +109,7 @@ func fromYAML(file string, s *state) (bool, string) {
 	if err = yaml.UnmarshalStrict([]byte(yamlFile), s); err != nil {
 		return false, err.Error()
 	}
-	addDefaultHelmRepos(s)
+	//addDefaultHelmRepos(s)
```
Comment (luisdavim) [inline internal/app/utils.go]: nit: if the function was removed there's no point in keeping this commented out here.

File: examples/example.toml (hunk 1 above)
Comment (sami-alajrami) [reply examples/example.toml]: copy-paste error :)

**Pull request merged**